package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mia-platform/lc39",
-  "version": "6.0.0",
+  "version": "6.0.1-rc.0",
```

Comment:
We don't use rc for libraries. It should stay 6.0.0

Reply:
The version is updated in the new commit.